README.md

```diff
@@ -37,12 +37,30 @@ Development workflow varies by developer, but a typical development experience m
  - make changes to scala code
  - try compiling (`~compile`) or running the service to inspect it (`~run`)
 
+#### Frontend Development
+
+To do frontend development you will want to install [`nvm`](https://github.com/creationix/nvm#install-script) and use a recent version of node (e.g. 4+). After following the directions above for starting the VM, start the API server and other backend services by running `./scripts/server`.
+
+Then _outside_ the VM, while the server is still running, run `npm start` while inside the `app-frontend/` directory. This will start a `webpack-dev-server` on port 9090 that will auto-reload after javascript and styling changes.
```

Comment:
Yeah. There doesn't appear to be much downside as far as I can tell -- we're not tied to it, but it makes frontend development easier without negatively impacting much else.

If we see issues developing across platforms we will have to re-assess I think.